ci-operator/step-registry/baremetalds/assisted/operator/ztp/baremetalds-assisted-operator-ztp-ref.yaml

```diff
@@ -15,5 +15,8 @@ ref:
   - name: ASSISTED_ZTP_CONFIG
     default: ""
     documentation: Configuration for the deploy_spoke_cluster.sh script. See https://github.com/openshift/assisted-service/blob/master/deploy/operator/README.md for more details on these settings.
+  - name: WAIT_FOR_ASSISTED_FULL_INSTALL_ZTP
```

Comment:
Where is it being used? Do you wish to pass it to the packet server?

Reply:
This is part of this work: https://github.com/openshift/assisted-service/pull/2551

Reply:
When defining a new variable on OpenShift CI it isn't automatucally available on the remote Packet server. it needs to be passed to that location

BTW instead of defining a new variable as part of the ref step definition, it can be set under ASSISTED_ZTP_CONFIG which is already exposed to the ZTP workflow on the Packet server. I honestly prefer this approach.

Reply:
I didn't know any of the above, thanks for sharing.

> BTW instead of defining a new variable as part of the ref step definition, it can be set under ASSISTED_ZTP_CONFIG which is already exposed to the ZTP workflow on the Packet server. I honestly prefer this approach.

This works for me, it sounds simpler and it still makes sense. Not sure what @lranjbar and @pawanpinjarkar prefer in this case.

Reply:
This is done in the PR for the new job here https://github.com/openshift/release/pull/21539/files#diff-c57a0b6469c4c70a7423ed9ae84e626bc59f0347557a3b8356ea1247c7c31ec7R320-R321 
This PR can be closed.